fedimintd/templates/params.html

```diff
@@ -14,20 +14,20 @@ <h3>Configure Federation</h3>
       <input type="text" class="form-control" id="federation_name" name="federation_name" placeholder="Cypherpunk Federation">
     </div>
     <div class="form-group">
-      <label for="bind_api">API Listen Address</label>
-      <input type="text" class="form-control" id="bind_api" name="bind_api" placeholder="127.0.0.1:8173" />
+      <label for="bind_p2p">P2P Listen Address</label>
+      <input type="text" class="form-control" id="bind_p2p" name="bind_p2p" placeholder="127.0.0.1:8173" />
     </div>
     <div class="form-group">
-      <label for="api_url">API Public URL</label>
-      <input type="text" class="form-control" id="api_url" name="api_url" placeholder="wss://api.your-website.com" />
+      <label for="p2p_url">P2P Public URL</label>
+      <input type="text" class="form-control" id="p2p_url" name="p2p_url" placeholder="wss://p2p.your-website.com" />
     </div>
     <div class="form-group">
-      <label for="bind_p2p">P2P Listen Address</label>
-      <input type="text" class="form-control" id="bind_p2p" name="bind_p2p" placeholder="127.0.0.1:8174" />
+      <label for="bind_api">API Listen Address</label>
+      <input type="text" class="form-control" id="bind_api" name="bind_api" placeholder="127.0.0.1:8174" />
```

Comment:
the word order in `bind_api` and `api_url` seems inconsistent

Reply:
I just copied variable names from DKG. Let me update both in follow-on PR.